### PR TITLE
[Case Print PDF] - Create PDF Layout (without Categories)

### PR DESCRIPTION
**Pull request by andvirga**
Jira: https://bugs.benetech.org/browse/CHI-514

Primary reviewer: @murilovmachado 

Changes description:
- Installed `react-pdf` package with dependencies.
- Created a PDF layout that is consumed by `react-pdf` to produce the final document. The starting point is `CasePrintView.tsx`
- Added CSS Styling to the PDF layout (the starting point is: `casePrint/styles/index.ts`
- Added `mockedData.ts` that contains all the mocked information that we want to display in the PDF (This will be replaced in a future story).
- Categories are not included in this layout.

Known issues and workarounds:
- Images (helpline logos) cannot be added to the repo and consumed directly with an import. since there is an issue with the last flex-plugins CLI update. The workaround consisted in upload these images to the S3 bucket.
- Child At Risk is using emojis to render the empty checkbox (⏹️) and the marked one (☑️), however, all square emojis are being cut off by half. I've opened a bug in ReactPDF repo (https://github.com/diegomura/react-pdf/issues/1132) Workaround: using the X cross emoji instead (❌).
- ChildLine ZA logo is rendered with a gray background, even when the image has a white background. No workaround so far.

PDF Previews:
[Case Print - ZA.pdf](https://github.com/tech-matters/flex-plugins/files/6064469/Case.Print.-.ZA.pdf)
[Case Print - ZM.pdf](https://github.com/tech-matters/flex-plugins/files/6064490/Case.Print.-.ZM.pdf)

### CODE REVIEW

File: plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  );
+};
+
+CasePrintSection.displayName = 'CasePrintSection';
+
+CasePrintSection.propTypes = {
+  sectionName: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  fieldValues: PropTypes.array.isRequired, // ToDo: put a better typing here
+};
```
Comment (murilovmachado) [inline plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx:45]: Do we need these `propTypes` lines?
We're already using `type OwnProps` for `sectionName` and `fieldValues`.

File: plugin-hrm-form/src/components/case/casePrint/CasePrintSummary.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      </View>
+    </View>
+  );
+};
+
+CasePrintSummary.displayName = 'CasePrintSummary';
+
+CasePrintSummary.propTypes = {
+  summary: PropTypes.string.isRequired,
+};
```
Comment (murilovmachado) [inline plugin-hrm-form/src/components/case/casePrint/CasePrintSummary.tsx:33]: Same here, I believe we don't need it.

File: plugin-hrm-form/src/HrmFormPlugin.js [hunk 3]
```diff
@@ -40,6 +40,8 @@ export const getConfig = () => {
   const featureFlags = manager.serviceConfiguration.attributes.feature_flags || {};
   const { strings } = manager;
   const definitionVersion = 'za-v1'; // will be moved to service configuration later on
+  const helplineLogoSource = 'https://tl-public-chat-za-staging.s3.amazonaws.com/za-logo.png'; // will be moved to service configuration later on
+  // const logoSource = 'https://tl-public-chat-zm-staging.s3.amazonaws.com/zm-logo.png';
```
Comment (murilovmachado) [inline plugin-hrm-form/src/HrmFormPlugin.js:44]: We can remove this line.

File: plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+              logoSource={helplineLogoSource}
+            />
+            <View>
+              <CasePrintDetails
+                status={caseDetails.status}
+                openedDate={caseDetails.openedDate}
+                lastUpdatedDate={caseDetails.lastUpdatedDate}
+                followUpDate={caseDetails.followUpDate}
+                childIsAtRisk={caseDetails.childIsAtRisk}
+                counselor={caseDetails.currentCounselor}
```
Comment (murilovmachado) [inline plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx:68]: souldn't it be `caseDetails.caseCounselor`?

**Review by murilovmachado — APPROVED** [review]
Looks good to me! Just one question: can this be merged with these mocked data in place?

File: plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx (hunk 1 above)
Comment (andvirga) [reply plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx:45]: You're right, I'll remove these PropTypes

File: plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx (hunk 4 above)
Comment (andvirga) [reply plugin-hrm-form/src/components/case/casePrint/CasePrintView.tsx:68]: Good catch! Thanks!

**Comment by andvirga** [comment]
> Looks good to me! Just one question: can this be merged with these mocked data in place?

I believe we can merge and deploy it to Aselo Development, but I'll ask Dee just in case.

**Pull request merged**